book/src/super-sql/patterns.md

```diff
@@ -30,13 +30,19 @@ and is documented in the
 When used in an expression, e.g., as a parameter to a function, the
 RE2 text is simply passed as a string, e.g.,
 ```
-regexp('foo|bar')
+regexp('foo|bar', this)
+```
+
+To avoid having to add escaping that would otherwise be necessary to
+represent a regular expression as a string, prefix with `r`, e.g.,
+```
+regexp(r'\w+(foo|bar)', this)
 ```
 
 But when used outside of expressions where an explicit indication of
 a regular expression is required (e.g., in a
 [`search`](operators/search.md) or 
-[`from`](operators/from.md) operator), the RE2 is instead
+[`from`](operators/from.md#database-operation) operator), the RE2 is instead
```

Comment:
From what I can see, the only place regexps are currently used with `from` is to match database pools, so I figured a subtle clarification would be to hyperlink to the relevant section.